### PR TITLE
[PLAT-98] Add feature flag for auto sub

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add feature flag for auto sub.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Locally by manually enabling on both web & mobile.

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/common/store/social/users/sagas.ts [hunk 1]
```diff
@@ -73,6 +77,15 @@ export function* followUser(
   yield* put(event)
 
   yield* call(confirmFollowUser, action.userId, accountId)
+  const shouldAutoSubscribe =
+    ((yield* call(getFeatureEnabled, FeatureFlags.AUTO_SUBSCRIBE_ON_FOLLOW)) as
```
Comment (dylanjeffers) [inline packages/web/src/common/store/social/users/sagas.ts:81]: Anyway this can be a little less nested?

**Review by dylanjeffers — APPROVED** [review]
Love this feature

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-plat-98

File: packages/web/src/common/store/social/users/sagas.ts (hunk 1 above)
Comment (raymondjacobson) [reply packages/web/src/common/store/social/users/sagas.ts:81]: kinda not sure what the best pattern is b/c the getFeatureEnabled stuff doesn't seem typed

i copied dis one https://github.com/AudiusProject/audius-client/blob/edc84337702102d4e460eaf508a24a1974f76949/packages/web/src/common/store/notifications/sagas.ts#L140

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-plat-98

File: packages/web/src/common/store/social/users/sagas.ts (hunk 1 above)
Comment (raymondjacobson) [reply packages/web/src/common/store/social/users/sagas.ts:81]: chatted offline -- will look into this but at a later date! agreed bad!!

**Pull request merged**